scripts/zones/Abyssea-Grauberg/npcs/qm_xibalba.lua

```diff
@@ -1,15 +1,15 @@
 -----------------------------------
 -- Zone: Abyssea-Grauberg
---  NPC: qm_xiabalba (???)
--- Spawns Xiabalba
+--  NPC: qm_xibalba (???)
```

Comment:
Please remove space before **NPC:** (so that it aligns with the start of the other lines)